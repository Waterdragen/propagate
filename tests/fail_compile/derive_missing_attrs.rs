use propagate::Propagate;

#[derive(Propagate)]
enum MyEnum {
    A,
    B,
}

fn main() {}
