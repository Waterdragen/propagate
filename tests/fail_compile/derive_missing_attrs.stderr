error: Enum must contain at least one `#[good]` or `#[bad]` attribute. Did you forget to mark a good or bad variant?
 --> tests/fail_compile/derive_missing_attrs.rs:3:10
  |
3 | #[derive(Propagate)]
  |          ^^^^^^^^^
  |
  = note: this error originates in the derive macro `Propagate` (in Nightly builds, run with -Z macro-backtrace for more info)
